imm.yaml

```diff
@@ -52,30 +55,27 @@
     Id: 145f2e3d-bb05-4ced-989b-7fb218fc6705
     Measures:
         -
-            Score: 0
-            Description: None
+            Score: 1
+            Description: 'Common scenarios'
 -
     Name: 'Date of Last IP Review'
     Id: da4ed776-0365-4d8a-a297-c4e91a14d646
     Measures:
         -
-            Score: 0
-            Description: None
-        -
+            Date: 2020-03-20
 -
     Name: 'Framework Version'
     Id: 6c0402b3-f0e3-4bd7-83fe-04bb6dca7924
     Measures:
         -
-            Score: 3
-            Description: 'Using the most current LTS version'
+            Framework: netcoreapp3.1
```

Comment:
The specs project is `netcoreapp3.1` but the main project is `netstandard2.0`

Reply:
Yes. This is how we do it. This rule indicates the latest version we support. We test against `netcoreapp3.1` because that's the version we support; we continue to target `netstandard2.0` so as not to cause problems for projects that tdepend on this and haven't yet upgraded to .NET Core 3.1.

But there's nothing about .NET Core 3.1 support that requires us to target .NET Core 3.1. Targetting .NET Standard 2.0 is a perfectly acceptable option. (It's what almost everything in Corvus does right now.)